wordpress-mobile/WordPress-iOS
# Reader: Configure `ReaderPostCardCell` with a `ReaderPost`

Part of #21645 
Merges into #21685

## Description

Configures the new reader card post UI with reader posts.

> **Note**
> The following will be in future PR's:
> - Button actions
> - Safe area fixes
> - Accessibility fixes

## Screenshots

| Light | Dark |
|------|------|
| ![light](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/83d7389f-3775-4ca4-bcbc-3936b7dc7bd1) | ![dark](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/a2ddd319-a466-48b4-9258-f445aaca704d) |

## Testing

To test:
- Launch Jetpack and login
- Enable the `Reader Improvements v1` feature flag if necessary
- Tap on Reader
- 🔎 **Verify**:
  - Cells are configured with reader post data
  - The appropriate controls appear depending on the blog
- Tap on `Likes` at the top
- 🔎 **Verify** liked posts show the correct button and image
- Tap on `Automattic`
- 🔎 **Verify**:
  - The avatar view appears with the author's image
  - The title has the author's name
  - The reblog button does not appear

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
